src/components/charge_points/openwb_satellite/chargePoint.vue

```diff
@@ -1,26 +1,28 @@
 <template>
-	<div class="charge-point-ipevse">
+	<div class="charge-point-satellite">
 		<openwb-base-text-input
 			title="IP oder Hostname"
 			subtype="host"
 			required
 			:model-value="configuration.ip_address"
 			@update:model-value="updateConfiguration($event, 'ip_address')"
 		/>
-		<openwb-base-number-input
-			title="Modbus-ID"
-			required
-			:min="1"
-			:max="255"
-			:model-value="configuration.modbus_id"
-			@update:model-value="updateConfiguration($event, 'modbus_id')"
+		<openwb-base-select-input
+			title="Zählertyp"
+			:options="[
+				{ value: 'sdm630', text: 'SDM630' },
```

Comment:
Der Klartext könnte etwas umfangreicher sein. Schließlich werden auch zum SMD 630 kompatible Zähler unterstützt. Aktuell z.B. der SDM 72DMv2.
```suggestion
				{ value: 'sdm630', text: 'SDM 630 / 72DMv2 und kompatible' },
```